src/login/logind-dbus.c

```diff
@@ -3169,6 +3169,8 @@ int match_job_removed(sd_bus_message *message, void *userdata, sd_bus_error *err
                         session->scope_job = mfree(session->scope_job);
                         (void) session_jobs_reply(session, unit, result);
 
+                        /* Scope job is done so leader must be gone as well. */
```

Comment:
I think this comment is misleading. `session_invalidate_leader()` does not assume that the leader is gone. Should this comment be "Check if the leader is actually gone." ?

Reply:
Maybe the only word that needs changing is "must". Instead we it should say "should".